backend/test/services/model.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { describe, expect, test, vi } from 'vitest'
+
+import { createModel, getModelById } from '../../src/services/v2/model.js'
+
+const mocks = vi.hoisted(() => {
```

Comment:
For readability and organisation could we use vi.hoisted to define a factory mock for each of our mocks rather than putting methods for different mocks in a single object called 'mocks'?